### ISSUE TITLE
Breaking change HA2025.01

### ISSUE THREAD

**Issue by djfanatix** [taarskog/home-assistant-component-somweb#21]
### System Health details

/

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [previous issues](https://github.com/taarskog/home-assistant-component-somweb/issues?q=is%3A+issue+label%3Abug)..

### Describe the issue

I have this in the logs, can you update this before 2025?
Many thanks

```
2024-06-07 11:35:56.977 WARNING (ImportExecutor_0) [homeassistant.components.cover] SUPPORT_CLOSE was used from somweb, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.CLOSE instead, please report it to the author of the 'somweb' custom integration
2024-06-07 11:35:56.981 WARNING (ImportExecutor_0) [homeassistant.components.cover] SUPPORT_OPEN was used from somweb, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.OPEN instead, please report it to the author of the 'somweb' custom integration
```

### Reproduction steps

1.
2.
3.
...

### Debug logs

```Text
/
```

**Comment by taarskog** [comment]
Yes. Thanks for reporting it. I've seen it and plan to update in the near future.

**Comment by taarskog** [comment]
Fixed in e10e69f
